src/rest/options/snapshots.ts

```diff
@@ -68,6 +68,41 @@ export interface IOptionsSnapshotContract {
   results?: SnapshotInfo;
 }
 
+// CF: https://polygon.io/docs/options/get_v3_snapshot_options__underlyingasset
+export interface IOptionsSnapshotChain {
+  status?: string;
+  request_id?: string;
+  previous_url?: string;
+  next_url?: string;
+  results?: SnapshotInfo[];
+}
+
+export interface IOptionsChainQuery extends IPolygonQuery {
+  contract_type?: string;
+  expiration_date?: string;
+  "expiration_date.lt"?: string;
+  "expiration_date.lte"?: string;
+  "expiration_date.gt"?: string;
+  "expiration_date.gte"?: string;
+  order?: "asc" | "desc";
+  limit?: number;
+  sort?: string;
```

Comment:
The accepted `sort` options should be `"ticker" | "expiration_date" | "strike_price"`